cgmodels/trailblazer/constants.py

```diff
@@ -0,0 +1,14 @@
+from enum import Enum
+
+
+class StrEnum(str, Enum):
+    def __str__(self) -> str:
+        return str.__str__(self)
+
+
+class AnalysisTypes(StrEnum):
```

Comment:
```suggestion
class AnalysisTypes(str, Enum):
```
Do we really need the StrEnum class?

Reply:
Maybe not. Let's try without it